coiote-aws-iot-cloud-formation/cloudFormation.json

```diff
@@ -16,6 +16,22 @@
             }
           ]
         },
+        "Policies": [
+          {
+            "PolicyName": "SecretsManagerForLambdaPolicy",
```

Comment:
Won't the names collide when we create the role the second time?

Reply:
https://stackoverflow.com/questions/57660653/how-to-create-some-random-or-unique-value-in-a-cloudformation-template